package/tests/integration/editor.spec.ts

```diff
@@ -46,5 +46,27 @@ test.describe('editor', () => {
             await page.keyboard.press('ArrowLeft');
             await expect(bracketElements).toHaveCount(2);
         });
+
+        test('wrap pasted extended datetime strings with datetime()', async ({ page }) => {
```

Comment:
what if I paste datetime
(2023-01-01T00:00:00Z)?

Would add a test that acoid double wrapping